### PR TITLE
Restrict payload size

### PR DESCRIPTION
**Pull request by mrsmkl**
Quick fix for https://github.com/celo-org/celo-monorepo/issues/1912
Next can check if it's already fixed properly in upstream.

Of course have to also check this actually fixes the problem we are having.

### CODE REVIEW

File: apps/explorer/lib/explorer/chain/events/publisher.ex [hunk 1]
```diff
@@ -22,7 +22,10 @@ defmodule Explorer.Chain.Events.Publisher do
 
   defp send_data(event_type) do
     payload = encode_payload({:chain_event, event_type})
-    @sender.send_notify(payload)
+
+    if byte_size(payload) < 7500 do
```
Comment (diminator) [inline apps/explorer/lib/explorer/chain/events/publisher.ex]: can we make this an ENV var?

File: apps/explorer/lib/explorer/chain/events/publisher.ex [hunk 2]
```diff
@@ -32,7 +35,10 @@ defmodule Explorer.Chain.Events.Publisher do
 
   defp send_data(event_type, broadcast_type, event_data) do
     payload = encode_payload({:chain_event, event_type, broadcast_type, event_data})
-    @sender.send_notify(payload)
+
+    if byte_size(payload) < 7500 do
```
Comment (diminator) [inline apps/explorer/lib/explorer/chain/events/publisher.ex]: ENV var

File: apps/explorer/lib/explorer/chain/events/publisher.ex (hunk 1 above)
Comment (diminator) [reply apps/explorer/lib/explorer/chain/events/publisher.ex]: or config

Comment (mrsmkl) [reply apps/explorer/lib/explorer/chain/events/publisher.ex]: Nope, it's a hardcoded value in postgres. Perhaps it could be a constant here.

**Pull request merged**